src/main.py

```diff
@@ -2,8 +2,11 @@
 import sys
 import pip
 import pkg_resources
+from datetime import date
+
 
 REQUIREMENTS_FILE_PATH = "requirements.txt"
+END_OF_SERVICE = date(2022, 4, 1)
```

Comment:
maybe april 3rd? it is still ok in the very first ithaca cycle since people will be paying the last hangzhou cycle

Reply:
people can modify the date themselves?